prog/vnet/subnet_nexus.rb

```diff
@@ -270,13 +270,6 @@ def nics_to_rekey
     nics_with_strand_label(%w[wait wait_setup]).all
   end
 
-  def update_stack_locked_nics(locked_nics)
```

Comment:
Similarly, could keep this method and have it call `update_stack`, as it also has multiple callers.